cmd/dep/vndr_importer.go

```diff
@@ -0,0 +1,153 @@
+// Copyright 2017 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bufio"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/golang/dep"
+	fb "github.com/golang/dep/internal/feedback"
+	"github.com/golang/dep/internal/gps"
+	"github.com/pkg/errors"
+)
+
+func vndrFile(dir string) string {
+	return filepath.Join(dir, "vendor.conf")
+}
+
+type vndrImporter struct {
+	logger  *log.Logger
+	verbose bool
+	sm      gps.SourceManager
+}
+
+func newVndrImporter(log *log.Logger, verbose bool, sm gps.SourceManager) *vndrImporter {
+	return &vndrImporter{
+		logger:  log,
+		verbose: verbose,
+		sm:      sm,
+	}
+}
+
+func (v *vndrImporter) Name() string { return "vndr" }
+
+func (v *vndrImporter) HasDepMetadata(dir string) bool {
+	_, err := os.Stat(vndrFile(dir))
+	return err == nil
+}
+
+func (v *vndrImporter) Import(dir string, pr gps.ProjectRoot) (*dep.Manifest, *dep.Lock, error) {
+	v.logger.Println("Detected vndr configuration file...")
+
+	packages, err := v.loadVndrFile(dir)
+	if err != nil {
+		return nil, nil, errors.Wrapf(err, "unable to load vndr file")
+	}
+
+	manifest := &dep.Manifest{
```

Comment:
Note to self: If #952 is merged first, we should use the constructor `dep.NewManifest`.